_ext/eclipse-cdt/gradle.properties

```diff
@@ -11,11 +11,10 @@ ext_group=com.diffplug.spotless
 ext_VER_JAVA=1.8
 
 # Compile dependencies
-VER_ECLIPSE_CDT=9.7
```

Comment:
I'm wondering if this was a bug in the eclipse-cdt release of 9.8, as it referenced 9.7

Reply:
Good catch!  I looked at it, and I believe you are correct.  I should publish a `9.8.1` bugfix.

Reply:
Thanks for the fix. I really missed to commit the vital change.